openshift/templates/nodejs.json

```diff
@@ -55,67 +64,65 @@
       "apiVersion": "v1",
       "metadata": {
         "name": "nodejs-example",
-        "labels": {
-          "name": "nodejs-example"
+        "annotations": {
+          "description": "Defines how to build the application"
         }
       },
       "spec": {
-        "triggers": [
-          {
-            "type": "GitHub",
-            "github": {
-              "secret": "secret101"
-            }
-          },
-          {
-            "type": "Generic",
-            "generic": {
-              "secret": "secret101"
-            }
-          },
-          {
-            "type": "ImageChange"
-          }
-        ],
         "source": {
           "type": "Git",
           "git": {
             "uri": "${SOURCE_REPOSITORY_URL}"
-          }
+          },
+          "contextDir": "${CONTEXT_DIR}"
         },
         "strategy": {
           "type": "Source",
           "sourceStrategy": {
             "from": {
               "kind": "ImageStreamTag",
-              "name": "nodejs:latest",
-              "namespace": "openshift"
-            },
-            "incremental": true
+              "namespace": "openshift",
+              "name": "nodejs:0.10"
+            }
           }
         },
         "output": {
           "to": {
-            "kind": "ImageStreamTag",
-            "name": "nodejs-example:latest"
+            "kind": "ImageStream",
+            "name": "nodejs-example"
           }
-        }
+        },
+        "triggers": [
+          {
+            "type": "ImageChange"
+          },
+          {
+            "type": "GitHub",
+            "github": {
+              "secret": "secret123"
```

Comment:
please make it so in every template. this is very important as the parameters are only way users can change this value when creating applications from UI